common/views/components/ArchiveTree/ArchiveTree.js

```diff
@@ -102,12 +123,12 @@ function createWorkPropertyFromWork(work) {
   };
 }
 
-function createNodeFromWork(work, openByDefault) {
+function createNodeFromWork(work, openStatus) {
```

Comment:
types? Makes it loads more easy to fathom to another developer as to what the function is expected to be doing.